app/poros/media_result.rb

```diff
@@ -2,18 +2,24 @@ class MediaResult
   attr_reader :id,
               :title,
               :type,
-              :audience_score,
-              :poster
+              :vote_average,
+              :poster,
+              :release_year
 
   def initialize(media_result_data)
     @id = media_result_data[:attributes][:id]
     @title = media_result_data[:attributes][:title]
-    @type = media_result_data[:attributes][:type]
-    @audience_score = media_result_data[:attributes][:audience_score]
-    @poster = media_result_data[:attributes][:poster]
+    @type = media_result_data[:attributes][:media_type]
+    @vote_average = media_result_data[:attributes][:vote_average]
+    @poster = media_result_data[:attributes][:poster_path] || media_result_data[:attributes][:poster]
+    @release_year = media_result_data[:attributes][:release_year]
   end
 
   def formatted_type 
     type.gsub("_", " ").titleize
   end
+
+  def round_vote 
+    vote_average.round(1)
+  end
 end
```

Comment:
Part of the problem with having two media poro's is we risk implementing changes in only one place. For example I could use this formatted_type method in my media poro, or I could forget to change the fields of Media audience score to vote average. There is a counter argument to be made for keeping them separate because maybe we display vote average for search results and audience score for the show page. We can discuss more as a group, but I am ok with having some of the Media fields just be 'nil' and having one poro.